crates/turbopack-core/src/environment.rs

```diff
@@ -223,6 +223,19 @@ impl Environment {
         })
     }
 
+    #[turbo_tasks::function]
+    pub async fn supports_commonjs_externals(self: Vc<Self>) -> Result<Vc<bool>> {
+        let this = self.await?;
+        Ok(match this.execution {
+            ExecutionEnvironment::NodeJsBuildTime(..) | ExecutionEnvironment::NodeJsLambda(_) => {
+                Vc::cell(true)
+            }
+            ExecutionEnvironment::Browser(_) => Vc::cell(false),
+            ExecutionEnvironment::EdgeWorker(_) => Vc::cell(true),
+            ExecutionEnvironment::Custom(_) => todo!(),
+        })
+    }
```

Comment:
```suggestion
    pub fn supports_commonjs_externals(&self) -> Vc<bool> {
        match self.execution {
            ExecutionEnvironment::NodeJsBuildTime(..) | ExecutionEnvironment::NodeJsLambda(_) => {
                Vc::cell(true)
            }
            ExecutionEnvironment::Browser(_) => Vc::cell(false),
            ExecutionEnvironment::EdgeWorker(_) => Vc::cell(true),
            ExecutionEnvironment::Custom(_) => todo!(),
        }
    }
```

would love a linter for this